Core/Utilities/TimeLeft/test/Test_TimeLeft.py

```diff
@@ -53,13 +93,27 @@ def test_getScaledCPU( self ):
       self.tl.runCommand = rcMock
 
       batchSystemName = '%sTimeLeft' % batch
-      batchPlugin = __import__( 'DIRAC.Core.Utilities.TimeLeft.%s' %
+      batchPlugin = __import__( 'DIRAC.Core.Utilities.TimeLeft.%s' % #pylint: disable=unused-variable
                                 batchSystemName, globals(), locals(), [batchSystemName] )
       batchStr = 'batchPlugin.%s()' % ( batchSystemName )
       tl.batchPlugin = eval( batchStr )
       res = tl.getScaledCPU()
       self.assertEqual( res, 300.0 )
 
+      for batch, retValue in [( 'MJF', MJF_ReturnValue )]:
+        self.tl = importlib.import_module( "DIRAC.Core.Utilities.TimeLeft.TimeLeft" )
+        rcMock = MagicMock()
```

Comment:
I think the importlib is dangerous, because this will change the module for _all_ tests hat follow this one (even in tests from other files when executed through py.test after this test). This might not cause problems for the module imported here, but if something more central is treated this way weird things can happen.